### ISSUE TITLE
<Epic> CAIA Support: mhv-medications

### ISSUE THREAD

**Issue by alexiawunder** [department-of-veterans-affairs/va.gov-team#58242]
## What does your team need support for? Check all that apply.
- [x] Launching one or more unauthenticated pages, online forms, apps, or tools on VA.gov
- [ ] Making revisions to one or more existing VA.gov pages
- [ ] Something else 
 
We want to include the content team as early as possible in the medications designs, as their feedback will be instrumental in guiding the high fidelity mocks. 

## Will this new product be released incrementally (for instance 25% of users initially)? 
  - [x] Yes
  - [ ] No
 
**Note:** If you checked yes, we'll reach out to discuss details about the content in the react widget.
  
## When do you expect to launch your product to 100% of users?
  TBD - We are still working under the assumption that we will have meds ready for phase 0 by October 2023.
  
## Supporting artifacts
*Please provide supporting artifacts as available.*

- [Product outline](https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/products/health-care/digital-health-modernization/mhv-to-va.gov/medications)
- [User flows](https://www.sketch.com/s/c117249d-4845-4740-a3e2-78ebde262de3/v/38Y4Qa/p/3B5DC627-0552-423B-9D05-1796643D4605/canvas?posX=-678.118645854775&posY=-685.7033198873772&zoom=0.3199818730354309)
- [Low Fidelity Sketch designs (all pages)](https://www.sketch.com/s/c117249d-4845-4740-a3e2-78ebde262de3/v/38Y4Qa/p/C5D940B0-048F-42DB-BBE7-A2D0F96C8FD5/canvas?posX=-10068.66851338589&posY=-664.0641904524819&zoom=0.05561014264822006)
- Link to any epics/issues that may be helpful:
  - [Prescription status definitions ](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/health-care/digital-health-modernization/mhv-to-va.gov/medications/research/Discovery/prescription%20%26%20refill%20statuses.md)
  - [MVP Requirements](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/health-care/digital-health-modernization/mhv-to-va.gov/medications/requirements/requirements.md)
  - [Pharmacy pages error messages
](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/health-care/digital-health-modernization/mhv-to-va.gov/medications/research/Discovery/Pharmacy_Pages_Error_Message_Analysis.xlsx)
  - [Mural Board for initial content mapping](https://app.mural.co/t/departmentofveteransaffairs9999/m/departmentofveteransaffairs9999/1681935491539/3add3a248ca8210f6cfd22f8a82f8c5579d8ee63?sender=u3196b80393a36f1ccebf7408)
## Will this work be going through the Collaboration Cycle?
- [x] Yes 
- [ ] No

## When does this work need to be done?
- Estimated launch date: TBD
- Estimated staging review date: TBD
- Content, Accessibility, and IA work needed by: **We would like to have content's feedback implemented before we begin high fidelity design work. Date TBD.** 

## Do you plan to bring this to an upcoming content office hours session?
- [ ] Yes, benefit content office hours (Thursdays, 3:00 p.m. to 3:45 p.m. ET)
- [ ] Yes, health  content office hours (Thursdays, 11:00 a.m. to 11:30 a.m. ET)
- [ ] Yes, unauth office hours (Mondays, 11:00 a.m. to 11:30 a.m. ET)
- [ ] No, but I'd like to schedule time to talk about this request
- [x] No, let's work asynchronously and meet if needed

**Note:** If we think this work would benefit from a collaborative session with you, we may ask you to bring it to office hours or set up a separate time to meet.

## About your team

- Team name: MHV Medications
-	OCTO-DE product owner:  Kay Lawyer and Lauren Alexanderson
-	Product manager: Kay Lawyer, Raquel Robertson
-	Designers: Alexia Wunder, Harrison Amelang, Coulton Bunney, Jenny Wang
-	FE engineer: Mark Dewey and Stephen Barrs
-	Product/team Slack channel: #mhv-medications-rx

## Next steps

**Once you’ve submitted this ticket, please post a link to this issue in the [#sitewide-content-ia](https://dsva.slack.com/channels/sitewide-content-ia) Slack channel and tag Randi Hecht.**

If you also need engineering support from the Public Websites team, fill out their [intake request form](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=Public+Websites%2C+Sitewide+content&labels=vsa-public-websites%2C+vsa%2C+vsa-public-websites-intake%2C+sitewide-content%2C+needs-grooming&template=public-websites-intake.md&title=%3CType+of+Request%3E+from+%3CTeam%3E).
  
If you need a page/URL redirected, a URL changed or a vanity URL set up, please submit a [Redirect, URL change, or vanity URL request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=mnorthuis&labels=ia&template=redirect-request.md&title=Redirect+Request)

```[tasklist]
### Tasks
```

**Comment by kristinoletmuskat** [comment]
@RLHecht @mnorthuis because of contracting issues (most of the MHV work was going to be done by Joni before she left), I can't work on this. So Mikki asked that I assign to her. Mikki let me know if there's any way I can support!

**Comment by laurwill** [comment]
Hi @alexiawunder @Hamelang @coultonbunney-usds ,
Just documenting here the approach around prescription statuses and dates for phase 0:
- We won't suggest changing the status terms at this time. Instead, we'll display these on detail pages only, with expandable components for the definitions of each status. On the more prominent list view, we'll display the actions Veterans can take (like filling or refilling a prescription) rather than the status (like "Active: Parked").
- We may suggest changing the date labels at this time. We'd like to avoid the need for "What does this date mean" supporting content.

**Comment by laurwill** [comment]
Hi @Hamelang and @alexiawunder , as I mentioned on Slack, I'm passing you clean versions of medications content for each page here, to update the prototype designs. Before we submit for midpoint, I'll either resolve and clean up the back and forth in the working content document or create a clean "frozen" version for platform team to refer to during midpoint. 

We may have a few more content changes based on ongoing SME convos or how it looks in the designs, but I'll try to keep that to a minimum. 

First up, landing page: [https://app.zenhub.com/files/133843125/22b2d280-e86e-40ef-8c73-5c2e1779f9af/download](https://app.zenhub.com/files/133843125/22b2d280-e86e-40ef-8c73-5c2e1779f9af/download)

**Comment by laurwill** [comment]
@Hamelang @alexiawunder here's all of the content with tracked changes showing, in case that's easier for you. I had to accept changes on the status content table because it was causing issues in Sharepoint. The rest is tracked.
[https://app.zenhub.com/files/133843125/da00d512-2ff2-4dcc-a8a8-71a44b066b81/download](https://app.zenhub.com/files/133843125/da00d512-2ff2-4dcc-a8a8-71a44b066b81/download)

I'll track any additional changes after this in the content source of truth in sharepoint.

**Comment by laurwill** [comment]
@Hamelang @alexiawunder please ignore the tracked changes to the sort options in that doc (I meant to remove that) — let's confirm those separately.

**Comment by laurwill** [comment]
@Hamelang @alexiawunder here's a clean version of the content document. I updated to include less overlap in the status definitions and "status-related content" that we're showing on the cards. This is to accommodate the approach from today's design collab session (displaying the "status-related content" and buttons just above the actual status field on detail pages)
[https://app.zenhub.com/files/133843125/ae24c13a-9f33-4fdb-a421-50ea59df3995/download](https://app.zenhub.com/files/133843125/ae24c13a-9f33-4fdb-a421-50ea59df3995/download)

**Comment by laurwill** [comment]
And here's the working content source of truth: https://dvagov-my.sharepoint.com/:w:/g/personal/laura_willwerth_va_gov/ET1vyapgp7JLlNeFvfI4FZcBjaEsP6TKZQU-OKXZq5eF3A?e=HFmi4E
